app/controllers/restaurants_controller.rb

```diff
@@ -0,0 +1,50 @@
+class RestaurantsController < ApplicationController
+  def index
+    @restaurant = Restaurant.all
+  end
+
+  def show
+    @restaurant = Restaurant.find(params[:id])
+  end
+
+  def new
+    @restaurant = Restaurant.new
+  end
+
+  def edit
+    @restaurant = Restaurant.find(params[:id])
+  end
+
+  def create
+    @restaurant = Restaurant.create(restaurant_params)
+    if @restaurant.save 
+      redirect_to root_path
+      flash[:notice] = "Saving was succefull"
+    else 
+      flash[:error] = "There was an error occured"
```

Comment:
🔨 @Alisher778 The directions called for the "happy path" approach, meaning we are assume there will be no errors, which means this conditional is not needed since (because we have no validations set, everything will be saved successfully). Flash messages are part of HW for Lesson 10 (not this hw assignment)